client/client_state.cpp

```diff
@@ -431,7 +431,7 @@ static void set_client_priority() {
 #ifdef __linux__
     char buf[1024];
     snprintf(buf, sizeof(buf), "ionice -c 3 -p %d", getpid());
-    system(buf);
+    if (!system(buf)) {}
```

Comment:
What's the purpose of this change? It basically changes nothing until we want to put smth between curly braces

Reply:
It removes a compiler warning